STEP_Project/tests/OUListContentTest.js

```diff
@@ -2,12 +2,64 @@ let tabl;
 let OUs;
 let numRows;
 
+describe('Testing the initButtons function', ()=>{
+    beforeEach(function()
+    {
+        tabl = document.createElement('table');
+        for (let i = 0; i < 3; i++)
+        {
+            tabl.insertRow();
+        }
+        OUs = tabl.rows;
+        numRows = OUs.length;
```

Comment:
Initialize numRows to 3 before and use it's value in the for loop while adding rows.

Reply:
Done